src/goto-checker/all_properties_verifier_with_trace_storage.h

```diff
@@ -12,10 +12,9 @@ Author: Daniel Kroening, Peter Schrammel
 #ifndef CPROVER_GOTO_CHECKER_ALL_PROPERTIES_VERIFIER_WITH_TRACE_STORAGE_H
 #define CPROVER_GOTO_CHECKER_ALL_PROPERTIES_VERIFIER_WITH_TRACE_STORAGE_H
 
-#include "goto_verifier.h"
-
 #include "bmc_util.h"
 #include "goto_trace_storage.h"
+#include "goto_verifier.h"
```

Comment:
This reformatting appears to be the result of adding and removing things in the process of work on this PR. If you removed all the changes to this file, then you would not need as main code-owners to approve...